modules/licenses.go

```diff
@@ -27,9 +27,12 @@ func findLicenseFile(dep *modfile.Require) ([]byte, error) {
 		return nil, fmt.Errorf("escaping the module version failed: %w", err)
 	}
 
+	modBasePath := path.Join(build.Default.GOPATH, "pkg/mod")
+	if os.Getenv("GOMODCACHE") != "" {
```

Comment:
```suggestion
	if val, ok := os.LookupEnv("GOMODCACHE"); ok {
```

https://pkg.go.dev/os#LookupEnv